runtime-light/component/component.cpp

```diff
@@ -6,15 +6,17 @@
 #include "runtime-light/core/globals/php-init-scripts.h"
 
 bool ComponentState::is_descriptor_stream(uint64_t update_d) {
-  if (opened_descriptors.contains(update_d)) {
-    return opened_descriptors[update_d] == DescriptorRuntimeStatus::Stream;
+  auto it = opened_descriptors.find(update_d);
```

Comment:
`if (const auto it = opened_descriptors.find(update_d); it != opened_descriptors.end()) ...`

Let's make no chance for errors :)